src/main/config/config_streamer.c

```diff
@@ -83,15 +78,19 @@ void config_streamer_start(config_streamer_t *c, uintptr_t base, int size)
     c->address = base;
     c->size = size;
     if (!c->unlocked) {
-#if defined(STM32F7) || defined(STM32H7)
+#if defined(EEPROM_IN_RAM)
```

Comment:
can you elaborate on what you require here?

Reply:
￼```
#if !defined(EEPROM_IN_RAM)
#if defined(STM32F7) || defined(STM32H7)	
        HAL_FLASH_Unlock();
#else
        FLASH_Unlock();
#endif
￼#endif
```

Reply:
I had it like that so that each case was clearer in the other PRs for the external flash and sd card code.  I'll change it for this PR as requested and will rebase the other PR's on this and go from there.

Reply:
ahh wait, never mind, i'm mistaken.

Reply:
I'll adjust some other similar #ifdef so it's all clearer too.